File: c++/cpp_hello/122.cpp
#include <iostream>
#include <math.h>
#include <vector>
using namespace std;

int main(){
    int n_son, count;
    cin >> n_son;

    vector<double> list(n_son);

    for (count = 0; count < n_son; count++){
        cin >> list[count];

    }
    double summa = 0, summa_kvadrat = 0, count_summa = 0;
    for (count = 0; count < n_son; count++){
        summa_kvadrat += pow(list[count], 2);
        summa += list[count];
        count_summa ++; 
    }
    cout << summa_kvadrat << endl << (summa / count_summa);

}

File: python/alogo_uz/122.py
from math import sin

n_son = int(input())
massiv = list(map(int, input().split()))

M = int(input())
summa = 0
summa_orta = 0
count = 0
for num in massiv:
    summa += pow(num, 2)
    summa_orta += num
    count += 1

print(summa, (summa_orta / count), end="\n")


